docker-compose.yml

```diff
@@ -1,8 +1,10 @@
 # NOTE:
 # If you update image version, make sure to update in docker-compose.ci.yml as well as
-# also do update GHA caches step in prbuild.yml
+# also do update GHA caches step in prebuild.yml
```

Comment:
Looks like spelling correction. Hehe. While at it, could you also help update better point to the full path i.e. `.github/workflows/prbuild.yml` i.e. PullRequest Build GH workflow

Reply:
Oh sorry.. my mistake